modules/react/_examples/stories/Layouts.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import {Meta, Story} from '@storybook/addon-docs';
+import {
+  AreaColumnPositioning,
+  CustomColumnPositioning,
+  CustomColumnWidth,
+  ResponsiveColumns,
+} from './examples/Layouts';
+
+<Meta title="Examples/Layouts" />
+
+# Canvas Kit Examples
+
```

Comment:
Adding a little content to avoid double headings / empty sections. 😄 
```suggestion
Below are layout examples to reference as you build your own. For more in-depth information on our `Grid` component, please reference our [`Grid` documentation](https://workday.github.io/canvas-kit//?path=/docs/components-containers-grid--basic).
```